src/deployer.js

```diff
@@ -0,0 +1,174 @@
+const { Setup, ContractInteract } = require('@openstfoundation/mosaic.js');
+const Web3 = require('web3');
+
+class Deployer {
+  constructor(ChainConfig) {
+    this.origin = {
+      web3: new Web3(ChainConfig.originWeb3Provider),
+      chainId: ChainConfig.originChainId,
+      deployer: ChainConfig.originDeployerAddress,
+      txOptions: {
+        gasPrice: ChainConfig.originGasPrice,
+      },
+      token: ChainConfig.simpleTokenAddress,
+      baseToken: ChainConfig.simpleTokenAddress,
+      burner: ChainConfig.originBurnerAddress,
+      masterKey: ChainConfig.originMasterKey,
+    };
+
+    this.auxiliary = {
+      web3: new Web3(ChainConfig.auxiliaryWeb3Provider),
+      chainId: ChainConfig.auxiliaryChainId,
+      deployer: ChainConfig.auxiliaryDeployerAddress,
+      txOptions: {
+        gasPrice: ChainConfig.auxiliaryGasPrice,
+      },
+      burner: ChainConfig.auxiliaryBurnerAddress,
+      masterKey: ChainConfig.auxiliaryMasterKey,
+    };
+  }
+
+  _deployOrganization() {
+    return Setup.organizations(
```

Comment:
note for mosaic.js: the organization setup should probably just be done for a singular organization on a given chain.

Reply:
For that there is `Organization.deploy()`